addons/resource.language.en_gb/resources/strings.po

```diff
@@ -18153,3 +18153,23 @@ msgstr ""
 msgctxt "#38025"
 msgid "Choose information provider"
 msgstr ""
+
+#: system/settings/rbp.xml
+msgctxt "#38026"
+msgid "Decode the stereo stream from 3D files"
```

Comment:
I was just wondering if we need this setting at all. I know that especially on the PI1 the additional load might be an issue, but thinking a bit about who will enable this setting and who likely won't, I came to the conclusion that we could use the "stereoscopic playback mode" setting for this. If it is set to "ignore", then also skip MVC parsing. If it's set to any other value, users likely want to watch 3D from time to time, and even if they prefer to watch in 2D by default, MVC parsing is needed to be able to switch into 3D mode. And because I doubt users that prefer watching stuff in 2D will turn this new setting on and off depending on their mood, it'll be always on anyways.
So do you agree that we could try to be smart about the need of MVC decoding and get rid of this setting, as I really doubt that 2D only users will buy 3D blu-rays or have other MVC material.